pkg/storage/allocator.go

```diff
@@ -55,6 +55,23 @@ var (
 	// making decisions based on them.
 	// Made configurable for the sake of testing.
 	MinLeaseTransferStatsDuration = time.Minute
+
+	// EnableLeaseRebalancing controls whether lease rebalancing is enabled or
+	// not. Exported for testing.
+	EnableLeaseRebalancing = envutil.EnvOrDefaultBool("COCKROACH_ENABLE_LEASE_REBALANCING", true)
```

Comment:
I was wondering about that too. There isn't much harm to leaving it in, but no one should really need to set it at this point. Think I should pull it out?